.clj-kondo/config.edn

```diff
@@ -13,7 +13,22 @@
                 test-helpers.unit/deftest-event      clojure.core/defn
 
                 taoensso.tufte/defnp                 clojure.core/defn}
- :linters      {:case-duplicate-test           {:level :error}
+ :linters      {:clojure-lsp/unused-public-var {:exclude-when-defined-by #{utils.re-frame/defn}
+                                                :exclude                 #{
+                                                                           ;; even if we are not
+                                                                           ;; using color now, we
+                                                                           ;; will need it later
+                                                                           quo.foundations.colors
```

Comment:
suggest to add the `src/native_module/core` ns here as well instead of removing them

Reply:
@clauxx, I think that in such case the code review should come into play. We simply shouldn't allow any new suppressions)

Reply:
But overall I see that warnings suppression looks incorrect for many people. I will remove the suppressions and lets try to quickly get rid of warnings in some nearest time :)

Reply:
> We simply shouldn't allow any new suppressions

can you elaborate why?

Reply:
@clauxx, as you wrote:
>  suppressing warnings might become a path of least resistance for CCs 

So as far as I understood you see as a problem that in newly commited code there will be more suppressed warnings. So if we are aiming for zero-warnings policy then we shouldn't allow any new warnings suppression to appear unless there is a very good reason for it. And this can be done at code review.